.github/workflows/sync.yml

```diff
@@ -32,4 +40,9 @@ jobs:
       env:
         SEAFOWL_PASSWORD: ${{ secrets.SEAFOWL_PASSWORD }}
       run: |
-        ./pants run src/s2sf:s2sf -- build https://seafowl-socrata.fly.dev
+        ./pants run src/s2sf:s2sf -- build https://seafowl-socrata.fly.dev || (echo "retrying"; sleep 10; ./pants run src/s2sf:s2sf -- build https://seafowl-socrata.fly.dev)
+    - name: Scale Fly VM down
+      if: always()
```

Comment:
Adding `always()` [appears](https://github.com/splitgraph/socrata-to-seafowl/actions/runs/3652913791/jobs/6171853934#step:9:1) to having the desired effect here 🎉